<?xml version="1.0" encoding="UTF-8"?>
<ui version="4.0">
 <class>AShapeHelpDialog</class>
 <widget class="QDialog" name="AShapeHelpDialog">
  <property name="geometry">
   <rect>
    <x>0</x>
    <y>0</y>
    <width>723</width>
    <height>518</height>
   </rect>
  </property>
  <property name="windowTitle">
   <string>Dialog</string>
  </property>
  <layout class="QVBoxLayout" name="verticalLayout">
   <item>
    <widget class="QLabel" name="label_3">
     <property name="text">
      <string>Reminder: CERN Root objects - configuration typically uses half sizes!</string>
     </property>
    </widget>
   </item>
   <item>
    <widget class="QListWidget" name="lwTypes"/>
   </item>
   <item>
    <widget class="QLabel" name="label">
     <property name="text">
      <string>Selected template:</string>
     </property>
    </widget>
   </item>
   <item>
    <widget class="QLineEdit" name="leSelected"/>
   </item>
   <item>
    <widget class="QLabel" name="label_2">
     <property name="text">
      <string>Help provided by ROOT:</string>
     </property>
    </widget>
   </item>
   <item>
    <widget class="QPlainTextEdit" name="pteHelp">
     <property name="maximumSize">
      <size>
       <width>16777215</width>
       <height>120</height>
      </size>
     </property>
     <property name="readOnly">
      <bool>true</bool>
     </property>
    </widget>
   </item>
   <item>
    <layout class="QHBoxLayout" name="horizontalLayout">
     <item>
      <spacer name="horizontalSpacer">
       <property name="orientation">
        <enum>Qt::Horizontal</enum>
       </property>
       <property name="sizeHint" stdset="0">
        <size>
         <width>40</width>
         <height>20</height>
        </size>
       </property>
      </spacer>
     </item>
     <item>
      <widget class="QPushButton" name="pbSelect">
       <property name="minimumSize">
        <size>
         <width>141</width>
         <height>0</height>
        </size>
       </property>
       <property name="text">
        <string>Use selected template</string>
       </property>
      </widget>
     </item>
     <item>
      <spacer name="horizontalSpacer_2">
       <property name="orientation">
        <enum>Qt::Horizontal</enum>
       </property>
       <property name="sizeHint" stdset="0">
        <size>
         <width>40</width>
         <height>20</height>
        </size>
       </property>
      </spacer>
     </item>
     <item>
      <widget class="QPushButton" name="pbClose">
       <property name="minimumSize">
        <size>
         <width>141</width>
         <height>0</height>
        </size>
       </property>
       <property name="text">
        <string>Close</string>
       </property>
      </widget>
     </item>
     <item>
      <spacer name="horizontalSpacer_3">
       <property name="orientation">
        <enum>Qt::Horizontal</enum>
       </property>
       <property name="sizeHint" stdset="0">
        <size>
         <width>40</width>
         <height>20</height>
        </size>
       </property>
      </spacer>
     </item>
    </layout>
   </item>
  </layout>
 </widget>
 <resources/>
 <connections/>
</ui>
